meshroom/ui/qml/main.qml

```diff
@@ -475,15 +490,26 @@ ApplicationWindow {
 
     // Utility functions for elements in the menubar
 
-    function initFileDialogFolder(dialog) {
-        if(_reconstruction.graph && _reconstruction.graph.filepath) {
-            dialog.folder = Filepath.stringToUrl(Filepath.dirname(_reconstruction.graph.filepath));
+    function initFileDialogFolder(dialog, importImages = false) {
+        let folder = "";
+
+        if (_reconstruction.graph && _reconstruction.graph.filepath) {
+            folder = Filepath.stringToUrl(Filepath.dirname(_reconstruction.graph.filepath));
+            if (imagesFolder.toString() === "" && workspaceView.imageGallery.grid.itemAtIndex(0) !== null) {
```

Comment:
We check `imagesFolder.toString() === ""` but it has not been initialized before.